charts/newrelic-prometheus-agent/CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### enhancements
+- adding support for `kube-dns` label for integration filters and `kube-dns` value to cover `coreDNS` use-case
```

Comment:
Nit: I would also mention adding `k8s-app` in source `source_labels` default values.

Reply:
thanks for noticing